docs/content/token/staking2/stakers.mdx

```diff
@@ -32,45 +32,105 @@ _Note: The corresponding **node operator** steps are outlined in a
 
 ## 1. Fetch node operator information
 
-TODO
-- Provide `operatorAddress, nodeID`
-- Uses `TH.04` script
+To fetch the node operator information for an address's staking proxy, the token holder must use use script TH.04:
```

Comment:
It'd be good to add a bit of context as to why this step is required. Something like this:

> After a node operator has agreed to run a node for you, they will create a `NodeInfo` object holding the IP address and public keys for your new node. This `NodeInfo` object will be stored in an account owned by the node operator.
> 
> The node operator should provide you with their account `address` and the `nodeID` for your new node. These parameters can be passed into the `get_node_info` script (`TH.04`) to confirm the node information.
>
> If the node information is correct, continue to step 2.

Reply:
Added. Thank you!